packages/graphiql-toolkit/src/storage/__tests__/base.spec.ts

```diff
@@ -1,7 +1,11 @@
 import { StorageAPI } from '../base';
 
 describe('StorageAPI', () => {
-  const storage = new StorageAPI();
+  let storage = new StorageAPI();
+
+  beforeEach(() => {
+    storage = new StorageAPI();
+  });
```

Comment:
these changes are not required but I thought it could be nice to have a clean slate in every test.

Reply:
thank you!